resolvers.js

```diff
@@ -190,7 +191,37 @@ const resolvers = {
 
     ...colorQuery,
   },
-
+  ProductsFilter: {
+    categories: parent =>
+      parent.categories.map(category => {
+        console.log('123');
```

Comment:
remove console.log and do this method with arrow function

Reply:
fixed